conf/defaults.ini

```diff
@@ -130,7 +130,13 @@ logging = false
 # This setting also applies to core backend HTTP data sources where query requests use an HTTP client with timeout set.
 timeout = 30
 
-# If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request, default is false.
+# How long the data proxy waits before sending a keepalive request. Default is 30 seconds.
+keep_alive = 30
+
+# How long the data proxy keeps an idle connection open before timing out. Default is 90 seconds.
+idle_conn_timeout = 90
```

Comment:
I would suggest renaming them to `keep_alive_seconds` and `idle_conn_timeout_seconds` as we do for most timeouts in the configuration (unfortunately not for the data proxy timeout as well).
This is useful since one is able to tell that the time unit is seconds by the setting name.

Reply:
Done!